src/Sulu/Component/Webspace/Tests/Unit/UrlTest.php

```diff
@@ -19,7 +19,7 @@ public function provideIsValidLocale()
     {
         return [
             ['de', 'at', 'de', 'at', true],
-            ['de', '', 'de', null, true],
+            ['de', '', 'de', '', true],
```

Comment:
@danrot here I'm not sure if I should change Url.php#L248 from `===` to `==` so this test case match. Before it did return true because empty($this->getCountry()) was true and it did never the equal check.